packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -787,7 +792,43 @@ export async function startBrowser(
     ...webdriverOptions,
     ...wdioOptions,
   })) as CompassBrowser;
-  await browser.navigateTo('http://localhost:7777/');
+
+  if (ATLAS_DOMAIN) {
+    // Navigate to a 404 page to set cookies
+    await browser.navigateTo(`https://${ATLAS_DOMAIN}/404`);
+
+    const cookiesFile = process.env.COOKIES_FILE;
+    if (!cookiesFile) {
+      throw new Error(
+        'ATLAS_DOMAIN is set but COOKIES_FILE is not. Please set COOKIES_FILE to the path of the cookies file.'
+      );
+    }
+    const cookies = JSON.parse(await fs.readFile(cookiesFile, 'utf8'));
```

Comment:
Eventually, the plan is to package `npm run test-web` as a container image, which can be pulled in MMS e2e task.

MMS already has a bunch of utilities for e2e test setup, in particular:
* Creating a new randomly generated test user
* logging in as that user
* Creating a new group
* Creating new clusters in that group

Thus, we probably don't need to reimplement all of those here. Instead, we can do those setup steps first, and then just save the cookies that are needed for authentication. Then, when we run the compass e2e test container, we just pass in the cookies, the group, and the names of the created clusters.